extensions/amp-story/1.0/amp-story.js

```diff
@@ -968,6 +969,7 @@ export class AmpStory extends AMP.BaseElement {
 
     const lastPage = this.pages_[this.getPageCount() - 1];
     if (activePage.element.hasAttribute(Attributes.ADVANCE_TO) ||
```

Comment:
maybe add a `TODO` to get rid of this once experiment is fully launched.

Reply:
As adding a todo for every experiment check might get repetitive, it's tracked here :) #20128

Reply:
sounds good.